test/_modules/Forks.t.sol

```diff
@@ -14,7 +15,7 @@ contract ForksTest is Test {
         ).run();
 
         if (res.stdout.length == 0) {
-            console.log("Skipping test because forking endpoint is not available");
+            rawConsoleLog("Skipping test because forking endpoint is not available");
```

Comment:
Maybe internally we use println when possible?

Reply:
Right, makes sense tho I think we should do it only when we are printing a `string` since `println(string,bytes)` is super heavy